### PR TITLE
Create front-end design backbone 

### PR DESCRIPTION
**Pull request by yeeteing**
Create front-end design backbone to ease other developers to start collaborate on the UI. 

This is how it looks like:
Screen Shot 2021-01-05 at 7 00 59 PM

### CODE REVIEW

**Review by hhdtan — COMMENTED** [review]
great skeletal structure to fire things up!

**Review by hhdtan — APPROVED** [review]

File: OnLand/ControlLaptop/angular-UI/src/app/components/navigation-bar/navigation-bar.component.css [hunk 1]
```diff
@@ -1,7 +1,53 @@
...
+    padding: 12px 0 8px 0;
+    flex-grow: 1;
+    flex-basis: auto;
+
+    font-style: normal;
+    font-weight: normal;
+    font-size: 18px;
+    text-align: center;
+    
+    background: linear-gradient(180deg, rgba(34, 40, 47, 1) 0%, rgba(129, 125, 125, 0) 100%);;
```
Comment (Shamima-Ali) [inline OnLand/ControlLaptop/angular-UI/src/app/components/navigation-bar/navigation-bar.component.css:50]: Double semicolons here in line 50.

Comment (Shamima-Ali) [inline OnLand/ControlLaptop/angular-UI/src/app/components/navigation-bar/navigation-bar.component.css:50]: This line has two semi colons

**Review by Shamima-Ali — COMMENTED** [review]
Looks good overall! :)

**Review by ShairYousuf — COMMENTED** [review]
Everything looks great!!

File: OnLand/ControlLaptop/angular-UI/src/app/components/navigation-bar/navigation-bar.component.css (hunk 1 above)
Comment (yeeteing) [reply OnLand/ControlLaptop/angular-UI/src/app/components/navigation-bar/navigation-bar.component.css:50]: good catch!

**Comment by ssun2019** [comment]
No problems on my side : D

**Review by colton-smith — APPROVED** [review]
Not much to say here, looks good!

**Pull request merged**